README.md

```diff
@@ -94,6 +98,17 @@ user.email = "test@test.com"
 updated_user = art.users.update(user)
 ```
 
+Update a user:
+```python
+from pyartifactory.models import User
+
+user = art.users.get("test_user")
+
+# Update user
+user.email = "test@test.com"
+updated_user = art.users.update(User(**user.dict()))
```

Comment:
With your version of Pydantic you have access to the `User.parse_obj` method, which is exactly for this use case: https://pydantic-docs.helpmanual.io/usage/models/#helper-functions

Reply:
Also, why do you need to create a new user from the old instance? Can't you just pass the modified `user`?

Reply:
Right, I can pass the pass the modified **user**